core/src/main/scala/org/broadinstitute/dsde/rawls/entities/compact/CompactEntityRepository.scala

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.dsde.rawls.entities.compact
```

Comment:
This seems like a reasonable way to be able to handle the transaction boundaries in the provider and still allow mocking.

Reply:
added a comment!